tools/check

```diff
@@ -277,6 +277,23 @@ run_build_runner() {
     check_no_changes "updates to *.g.dart files" '*.g.dart'
 }
 
+run_l10n() {
+    local output_files=lib/generated/l10n/zulip_localizations'*'.dart
```

Comment:
nit: I don't love calling this "files" — it makes it sound like it's an actual path (or list of actual paths) to some files. As mentioned at https://github.com/zulip/zulip-flutter/pull/1058#discussion_r1847653173 , I'd like to make explicit that this is instead a Git pathspec.

This would do it:
```suggestion
    local output_pathspec=lib/generated/l10n/zulip_localizations'*'.dart
```